pkg/kp/kv.go

```diff
@@ -50,6 +62,42 @@ func (err KVError) Error() string {
 	return fmt.Sprintf("%s failed for path %s", err.Op, err.Key)
 }
 
+func (c consulStore) PutHealth(res WatchResult) (time.Time, time.Duration, error) {
+	key := healthPath(res.Service, res.Node)
+
+	t, value := addTimeStamp(res)
```

Comment:
Is it okay for clients to pass a `Time` on the watch result? It looks like this will always override it. Probably worth saying that somewhere or adding a guard clause.